pkgs/applications/science/electronics/verilator/default.nix

```diff
@@ -1,22 +1,21 @@
 { lib, stdenv, fetchurl
-, perl, flex, bison
+, perl, flex, bison, python3
 }:
 
 stdenv.mkDerivation rec {
   pname = "verilator";
-  version = "4.100";
+  version = "4.108";
 
   src = fetchurl {
     url    = "https://www.veripool.org/ftp/${pname}-${version}.tgz";
-    sha256 = "0vg1gk1hqlnz74gfpf57588758myxvhqzi37yl4vqjcq40r83nr2";
+    sha256 = "00i7am41w9v4smhl64z7s95wdb55f684y89mc0hbc07j1ggc33lf";
   };
 
   enableParallelBuilding = true;
   buildInputs = [ perl ];
-  nativeBuildInputs = [ flex bison ];
+  nativeBuildInputs = [ flex bison python3 ];
 
-  # these tests need some interpreter paths patched early on...
-  doCheck = false;
```

Comment:
This needs to be reverted, because the tests still seem to fail.

Reply:
The error looks like it might be related to this old issue ... https://github.com/NixOS/nix/issues/1205

I'm going to try [this](https://nixos.wiki/wiki/Packaging/Quirks_and_Caveats#Package_simple_python_scripts) workaround.  I'll revert if that still doesn't work.

Reply:
It still failed, so I reverted the `doCheck` back to `false`